### DISCUSSION TITLE
WHIP-14: Dual token model

### DISCUSSION THREAD

**Discussion by ballandiesmark** [wihi-labs/WHIP#22] [Ideas]
Thank you @urosvk for putting this together. I can see this forming in front of my eyes. Im very excited.

Some comments in the following. In case I get your go, I could incorporate those comments in the document and you could revise afterwards (but feel free to execute yourself, etc.):

- We need to make clear that the tokens represent voting rights, but hat the initiation of releasing funds from the treasury is not directly linked to the token but has to be initiated by the governance committe or an appointed commission. We can (and Im in favor of it) make all releases from the treasury subject to a vote by the DAO. Hence, the initiative right would lie with the governance committee/ a comission, but the decision/ approval would lie with the DAO (similar how budgets are approved in democracies) - This would need to be applied to both, the core treasury and the supporter treasury.

- I would suggest to directly create a supporter syndicate commission consisting of three individuals that want to be more active (potentially accompanied by one core team member in the beginning). This would empower supporters right from the beginning to talk/ negotiate with the DAO. Also, it would decentralize responsibility to maintain/ manage the supporter community to that commission, freeing capacity from the core team. In order to implement this, we would require a resolution from he governance commitee and an approval in a mail-in-ballot or general assembly of association members
- What is the reason to restrict funds in treasury to only be of WNT? [read it later in the rationale, I think this makes sense]
- What do you mean by swap? I understand that tokens of WNT type can be send to treasury and sender receives an amount of another type of token, right? Conditions defined by the 80% vote?  Following above reasoning, I would suggest that conditions are identified/ proposed by governance committe (or a responsible commission) and approved by DAO. Potentially, just getting the thought, we could also have a second mechanism which empowers the DAO to make an initiative, but the governance committee needs to approve (hence having the two mechanisms: a) initiative by gov committee requires approval by DAO, b) initiative by DAO requires approval by GOV committee.
- WiHi are minted based on the conditions defined by the “supporter syndicate commission” right?
- Supporter treasury: could it make sense to allow anyone to fund it with WNT tokens? Would decentralize the funding of it (potentially a vehicle for whales to give something back, etc.). Have the feeling that this increases the flexibility without impacting its function negatively.
- some literature suggesting to increase the dimensionality in tokens (could be referenced in rationale):
-- A “Social Bitcoin” could sustain a democratic digital world https://link.springer.com/article/10.1140/epjst/e2016-60156-7 
-- To Incentivize or Not: Impact of Blockchain-Based Cryptoeconomic Tokens on Human Information Sharing Behavior https://ieeexplore.ieee.org/abstract/document/9825658 
-- Finance 4.0 - Towards a Socio-Ecological Finance System https://library.oapen.org/handle/20.500.12657/49519

**Comment by ballandiesmark** [comment]
- In order to improve information flows, one could require for the governance committee meetings and/ or supporter syndicate meetings that an observer of the other body participates (who could provide inputs into the discussion and could share the information with her/ his body). The observer does not have voting rights, etc. [e.g. the observer could be also part of the committees communication channels on discord, etc.]

**Comment by urosvk** [comment]
I've incorporated all your suggestions and/or made explanatory statements when necessary, save for one point: I think the supporter syndicate, insofar as one should exist, should be created as part of another WHIP. I would encourage the supporter layer to initiate this WHIP so that it give feedback on how it may organize itself. Until such a time, I would expect discussions regarding funding of the supporter treasury to be done on an ad hoc basis within the DAO.

**Comment by ballandiesmark** [comment]
I like the changes and think that the proposal is basically good to go. 

However, one thing that could cause confusion for a non-expert reader: The mechanism of the "swap" probably needs to be explained explicitly and early in the proposal. It is mentioned for the first time in the last bullet point of the WNT - but could be understood as something like the swap in a burn-and-mint token model [in particular, if someone with such a mindset reads this WHIP]. As I understand it, the swap in this case is a mechanism that gives a token holder the guaranteed right to exchange Token A units for a share in a treasury that is directly proportional to the proportion of Token A units exchanged to all existing Token A units. Moreover, the exchanged Token A units are burned so that the remaining token A units have an equal access to the treasury as the burned token units. In addition, the treasury underlying the Token A are locked/ once made accesible by Token A cannot be removed in another way then swapping them. In other words, one can say that one source of value of Token A is the associated treasury.

Moreover, it might be beneficial to illustrate in the motivation Heliums token model with the two subdaos and potential scenarios of the relationship between the WNT and WiHi token. As effectively communicated by you in our meeting, WiHi might trade higher on a open market than the price guaranteed by the swap mechanism, similar to Heliums Mobile token, because the market wants to incentivize miners to build out the mobile network. This basically might/ will mitigate the fact that the supporter treasury is empty in the beginning. 

Also, in the motivation we could mention your valuable thinking that using a utility token like WiHi instead of points is a more fair, transparent and regulatory compliant mechanism.
